upgrade-instructions.md

```diff
@@ -28,8 +28,26 @@ Change list:
  - [Hybrid blobStore replaces Union blobStore](#hybrid-blobstore-replaces-union-blobstore)
  - [New forbidden set of characters in Usernames local part](#new-forbidden-set-of-characters-in-usernames-local-part)
  - [UidValidity and maildir](#uid-validity-and-maildir)
+ - [UidValidity and JPA or Cassandra](#uid-validity-and-jpa-or-cassandra)
  
-### UidValidity and maildir
+### UidValidity and JPA or Cassandra
+
+Date 26/02/2020
+
+SHA-1 XXX
+
+JIRA: https://issues.apache.org/jira/browse/JAMES-3074
+
+Concerned products: JPA mailbox backend, Cassandra mailbox backend
+
+Non-Maildir backends could generate '0' uid validity with a low probability, which is invalid regarding RFC-3501. 
```

Comment:
Looking at the code, we actually generated negative numbers too. So it's 1 bad out of 2 uidvalidity instances, right?

Reply:
Theres a math.abs